javascript/implementation/Bundler.ts

```diff
@@ -89,7 +89,7 @@ export class Bundler implements BundleView {
      * @param bundleInfo the bundle metadata to add when serializing
      * @returns serialized
      */
-    seal(bundleInfo: BundleInfo, keyPair: KeyPair): void {
+    seal(bundleInfo: BundleInfo, keyPair: KeyPair, prior_hash?: Bytes): void {
```

Comment:
What is the reason for using snake case here?

Reply:
Too much context switching between python and Javascript/Typescript.